minidns-core/src/main/java/de/measite/minidns/DNSClient.java

```diff
@@ -154,4 +125,20 @@ public static synchronized void addDnsServerLookupMechanism(DNSServerLookupMecha
     public static synchronized boolean removeDNSServerLookupMechanism(DNSServerLookupMechanism dnsServerLookup) {
         return LOOKUP_MECHANISMS.remove(dnsServerLookup);
     }
+
+    public boolean isAskForDnssec() {
+        return askForDnssec;
+    }
+
+    public void setAskForDnssec(boolean askForDnssec) {
+        this.askForDnssec = askForDnssec;
+    }
+
+    public boolean isDisableResultFilter() {
+        return disableResultFilter;
+    }
+
+    public void setDisableResultFilter(boolean disableResultFilter) {
+        this.disableResultFilter = disableResultFilter;
+    }
```

Comment:
How about resultFilteringEnabled (isResultFilteringEnabled / setResultFilteringEnabled)?
I'm not sure about askForDnssec. Is this rooted in the RFCs? Would dnssecEnabled make sense?